lesson_03/maps/maps.go

```diff
@@ -0,0 +1,22 @@
+package maps
+
+import (
+	"fmt"
+	"sort"
+)
+
+func SortKeys(mapa map[int]string) []int {
```

Comment:
mapa is not the best name for argument) in small functions you can define variables just with one letter, m  in this case